DEV-1.java

```diff
@@ -0,0 +1,11 @@
+public class NumbersOnScreen {
+  public static void main(String[] args) {
+    for(int i = 0; i < 101; i++) {
+      if (i % 3 == 0) {
+        System.out.println("3*" + i/3);
+        i++;
```

Comment:
avoid to increment cycle counter in such non-obvious way as it always leads to logical errors